cli/context/store/store.go

```diff
@@ -18,15 +18,41 @@ import (
 
 // Store provides a context store for easily remembering endpoints configuration
 type Store interface {
+	StorageInfo
+	Write
+}
+
+// Read provides read-only (without list) access to context data
+type Read interface {
```

Comment:
Nit: What about using the same terminology than io.Writer/Reader... Lister etc..?